tests/func/test_update.py

```diff
@@ -172,3 +172,35 @@ def test_update_import_url(tmp_dir, dvc, tmp_path_factory):
 
     assert dst.is_file()
     assert dst.read_text() == "updated file content"
+
+
+def test_update_git_tracked(tmp_dir, dvc, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.scm.repo.index.remove([".dvc"], r=True)
```

Comment:
what's the need of removing `.dvc` from the index itself? :thinking:

Reply:
The purpose of this issue is to check that we can `dvc import` and then `dvc update` from a git repository. So I'm purging DVC and committing that. My reasoning is that when the repo is cloned so that it can be imported or updated, its `.dvc` directory being in the index makes a difference.

Reply:
Using new `git_dir` dir helper now.